x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_tables.tsx

```diff
@@ -214,11 +213,11 @@ export const RulesTables = React.memo<RulesTableProps>(({ selectedTab }) => {
   }, [rules, isAllSelected, setIsAllSelected, setSelectedRuleIds]);
 
   const isTableEmpty =
-    !isPrepackagedStatusLoading &&
-    prePackagedRulesStatus?.rules_custom_installed === 0 &&
-    prePackagedRulesStatus.rules_installed === 0;
+    !isRulesInfoLoading &&
+    rulesInfo?.rules_custom_count === 0 &&
+    rulesInfo?.rules_prebuilt_installed_count === 0;
 
-  const shouldShowRulesTable = !isPrepackagedStatusLoading && !isLoading && !isTableEmpty;
```

Comment:
Can we show the table without waiting for `isRulesInfoLoading`?

Reply:
In some low probability scenarios filters can load after the table data though this filters data isn't critical for displaying the table so `isRulesInfoLoading` can be removed.